docs/specification/1.0.md

```diff
@@ -45,7 +53,7 @@ curl "https://example.com/cds-services"
 }
 ```
 
-Developers of CDS Services must provide a well-known endpoint allowing the EHR to discover all available CDS Services, including information such as the purpose of the CDS Service, when it should be invoked, and any data that is requested to be prefetched.
+Developers of CDS Services MUST provide a stable endpoint for allowing EHRs to discover available CDS Services, including information such as the purpose of the CDS Service, when it should be invoked, and any data that is requested to be prefetched.
```

Comment:
This sentence just needs to be pulled up to the top of the `Discovery` section. When we were using another documentation templating engine (Slate), our code examples had to proceed the text that it was in reference to. With the switch to MkDocs, our examples are now in-line which causes this to look odd.

Moving this sentence to the top of the section will make the new sentence you added above redundant I believe.

Out of curiosity, why do you like "stable" more than "well-known"? The word "well-known" to me better describes why we are prescriptive of the `/cds-services` endpoint. The word "stable" to me only means that an endpoint must be accessible/available rather than at some predictable location.